webapp/app/static/css/base.css

```diff
@@ -691,6 +691,11 @@ footer a{
 .form-row-center {
     display: flex;
     align-items: baseline;
+    margin: 0 calc(-1 * var(--spacing-02));
```

Comment:
A comment would help me a lot. Isn't this pushing it across the borders of the column?